Dockerfile

```diff
@@ -3,20 +3,33 @@ FROM ${BUILD_FROM}
 
 SHELL ["/bin/ash", "-o", "pipefail", "-c"]
 
-ARG MDNS_REPEATER_VERSION
 RUN \
     apk add --no-cache --virtual .build-deps \
         build-base \
         git \
-    \
-    && git clone -b ${MDNS_REPEATER_VERSION} --depth 1 \
+        linux-headers
```

Comment:
Each layer need to install build tools and remove that afterward for layer caching: https://github.com/home-assistant/docker/blob/master/Dockerfile#L57 - https://github.com/home-assistant/docker/blob/master/Dockerfile#L82